gen/dcos-config.yaml

```diff
@@ -211,9 +211,41 @@ package:
       }
 {% case "false" %}
 {% endswitch %}
+  - path: /etc_master/logrotate_master.config
+    content: |
+      compress
+      delaycompress
+      notifempty
+      missingok
+      nomail
+      /var/lib/dcos/mesos/log/* {
```

Comment:
From our in-person sync: Change this to point to the leading log file.  It doesn't make sense to (potentially) rotate random files in the directory anymore.